server/test/auth.test.js

```diff
@@ -0,0 +1,21 @@
+require("dotenv").config();
+
+const chai = require("chai");
+const chaiHttp = require("chai-http");
+const { app } = require("../app.js");
+
+chai.should();
+chai.use(chaiHttp);
+
+describe("/GET logout", () => {
```

Comment:
Hmmm... having a logout path that uses `GET` doesn't sound right. Have you given this any thoughts? (this [article](https://www.baeldung.com/logout-get-vs-post) might be a good read)

We can change the route later but we should setup tests independently from the implementation (this isn't always easy when both are done by the same developers 😢).

Reply:
Awesome article! It was informative to learn how modern web accelerators can prefetch "GET" requests for faster loading. Since my other ticket #2 is a superset to this, I have modified the HTTP request to type "POST" on both client and server-side for its #63. Thanks for pointing this out!